src/core/Akka/Actor/ActorCell.Children.cs

```diff
@@ -18,7 +18,7 @@ namespace Akka.Actor
     public partial class ActorCell
     {
         private volatile IChildrenContainer _childrenContainerDoNotCallMeDirectly = EmptyChildrenContainer.Instance;
-        private long _nextRandomNameDoNotCallMeDirectly;
+        private long _nextRandomNameDoNotCallMeDirectly = -1; // Interlocked.Increment automatically adds 1 to this value. Allows us to start from 0.
```

Comment:
Cluster router uses AtomicCounter for this purposes. Could we use it here. Or it has some drawbacks?